src/main/java/org/aksw/simba/lemming/RemoveEdgeDecorator.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * 
+ */
+package org.aksw.simba.lemming;
+
+import it.unimi.dsi.fastutil.ints.IntArrayList;
+import it.unimi.dsi.fastutil.ints.IntSet;
+
+/**
+ * 
+ * Concrete Decorator class to simulate removing of an edge to the ColouredGraph
+ * object in the current iteration
+ * 
+ * @author Pranav
+ *
+ */
+public class RemoveEdgeDecorator extends ColouredGraphDecorator {
+
+    /**
+     * 
+     */
+    public RemoveEdgeDecorator() {
+        super();
+    }
```

Comment:
Again, a decorator without anything to decorate typically does not make much sense :thinking:

Reply:
Done